output_folder: ""
communication_range: 2
robots:
  - name: rb1
    rviz: no
    informe_initial_pose: true
  - name: rb2
    rviz: no
    informe_initial_pose: true